### ISSUE TITLE
uncertainty argsort wrong order

### ISSUE THREAD

**Issue by sumukhaithal6** [Mephisto405/Learning-Loss-for-Active-Learning#6]
Shouldn't we sort the uncertainty in descending order?(based on the paper)
The most uncertain ones needs to be sent to the oracle.
However in the code , np.argsort selects the sample in ascending order

**Comment by Mephisto405** [comment]
See the following lines (268~267 of main.py);
```
# Update the labeled dataset and the unlabeled dataset, respectively
labeled_set += list(torch.tensor(subset)[arg][-ADDENDUM:].numpy())
unlabeled_set = list(torch.tensor(subset)[arg][:-ADDENDUM].numpy()) + unlabeled_set[SUBSET:]
```

I sorted the uncertainty in ascending order and append the "rightmost" elements to the labeled_set. So I actually put the most uncertain elements :)

**Comment by sumukhaithal6** [comment]
Oh Okay, Thanks for your reply!
